Shopify/polaris-viz
# Upgrade to loom v1

## What does this implement/fix?







This change upgrades `loom` and related packages from the pre-v1 versions to v1.

V1 is the latest version available and contains some API changes to `loom.config.ts`.

## Does this close any currently open issues?

No

## What do the changes look like?

There are no code changes in the built output at all!

Here's how to verify:

1. On the `master` branch, run build: `yarn run build`
2. Move the build folder somewhere else (e.g. `mv build /tmp/build-master-branch`)
3. Checkout this branch: `git fetch origin && git checkout upgrade-to-loom-1 && dev up`
4. Run build: `yarn run build`
5. Compare the builds: `diff -ru -x '*.tsbuildinfo' /tmp/build-master-branch build` and see that this returns nothing
 
## Storybook link

N/A

### Before merging

- 🚫 Check your changes on a variety of browsers and devices.
- 🚫 Update the Changelog's Unreleased section with your changes.
  - Since nothing actually changed, I don't think we need a new CHANGELOG entry, but I'm more than happy to add one if people want.
- 🚫 Update relevant documentation, tests, and Storybook.
